src/core/env/dev/terraform.tfvars

```diff
@@ -35,6 +35,12 @@ cidr_subnet_pnpg_cosmosdb_mongodb = ["10.1.140.0/24"] #this is a place holder fo
 cidr_subnet_private_endpoints     = ["10.1.141.0/24"]
 cidr_subnet_load_tests            = ["10.1.142.0/24"]
 
+#
+# Pair VNET
+#
+cidr_pair_vnet                = ["10.2.0.0/16"]
+cidr_subnet_pair_dnsforwarder = ["10.2.134.0/29"]
```

Comment:
🔥 The Pair Vnet must be `10.101.0.0/16`

Reply:
Ok, but I followed your PR (https://github.com/pagopa/cstar-infrastructure/pull/900/files#diff-034351e60fc22d293ae9d622a94fd5790ec583200c69ded2d482a50a5e253551) and here the IP is *.2.*.* so why?
Did you modify it afterward?